src/MarketingPageAcceptanceTests.TestData/Solutions/CreateSolution.cs

```diff
@@ -27,8 +28,21 @@ public static Solution CreateNewSolution(string prefix = "Auto")
 
         private static string RandomSolId(string prefix)
         {
+            var faker = new Faker();
+            List<string> randomChars = new List<string>();
+            for (int i = 0; i < 10; i++)
+            {
+                randomChars.Add(faker.Random.AlphaNumeric(1));
+            }
+
+            var randomChar = Information.RandomInformation.GetRandomItem(randomChars).ToString();
+
             string timestamp = DateTimeOffset.Now.ToUnixTimeMilliseconds().ToString();
-            return prefix + timestamp.Substring(timestamp.Length - 10);
+            timestamp = timestamp.Substring(timestamp.Length - prefix.Length);
+
+            string totalString = prefix + timestamp;
+
+            return totalString.Length > 13 ? totalString.Substring(0, 13) + randomChar : totalString + randomChar;
```

Comment:
For readability sake, can you move the first loop and the randomChar into it's own method with a name which clearly explains what it's doing.

Same for the middle bit, and same for last bit